src/plugins/auto/auto_schedule.cpp

```diff
@@ -176,14 +227,19 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
             _loadContext[CPU].isEnabled = false;
         }
     }
+    _loadContext[FALLBACKDEVICE].usefeature = false;
```

Comment:
this is for a switch from config for use to control right? are we gonna to introduce a property?

Reply:
Yes, this is a switch to _loadContext[FALLBACKDEVICE] feature, Add this switch because I want to use the original _loadContext[i].task. But _loadContext[FALLBACKDEVICE] can not use 'future'